lib/libs/email/content/respondToRai/emailTemplates/ChipSpaCMS.tsx

```diff
@@ -7,30 +7,27 @@ import {
 } from "../../email-components";
 import { BaseEmailTemplate } from "../../email-templates";
 
-export const ChipSpaCMSEmail = (props: {
+export const ChipSpaCMSEmail = ({
+  variables,
+}: {
   variables: Events["RespondToRai"] & CommonEmailVariables;
-}) => {
-  const { variables } = props;
-  const previewText = `CHIP SPA ${variables.id} RAI Response Submitted`;
-  const heading = "The OneMAC Submission Portal received a CHIP SPA RAI Response Submission";
-  return (
-    <BaseEmailTemplate
-      previewText={previewText}
-      heading={heading}
-      applicationEndpointUrl={variables.applicationEndpointUrl}
-      footerContent={<BasicFooter />}
-    >
-      <LoginInstructions appEndpointURL={variables.applicationEndpointUrl} />
-      <PackageDetails
-        details={{
-          "State or Territory": variables.territory,
-          Name: variables.submitterName,
-          "Email Address": variables.submitterEmail,
-          "CHIP SPA Package ID": variables.id,
-          Summary: variables.additionalInformation,
-        }}
-      />
-      <Attachments attachments={variables.attachments} />
-    </BaseEmailTemplate>
-  );
-};
+}) => (
+  <BaseEmailTemplate
+    previewText={`CHIP SPA ${variables.id} RAI Response Submitted`}
```

Comment:
```suggestion
    previewText={`CHIP SPA RAI Response for ${variables.id} Submitted`}
```
https://confluenceent.cms.gov/display/MACPRO3/Email+Notifications+for+Package+Actions#EmailNotificationsforPackageActions-ResponsetoCHIPSPARAI

assuming this is still valid? 😆

Reply:
Resolving with https://github.com/Enterprise-CMCS/macpro-mako/pull/1117/commits/83e2e2471ea17a68a61e2e1cdedbb65ab51bca35